pkg/metrics/store_test.go

```diff
@@ -19,6 +19,7 @@ package metrics
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please configure goimports to format code

Reply:
I'm sorry, I will do this